docs/src/configuration.rst

```diff
@@ -35,14 +35,21 @@ To provide the ``carta`` user with these privileges, you must make modifications
 Authentication
 ~~~~~~~~~~~~~~
 
-When configured to use PAM or LDAP authentication, the controller signs and validates refresh and access tokens with SSL keys. You can generate a private/public key pair in PEM format using ``openssl``:
+The controller signs and validates tokens with SSL keys. You can generate a private/public key pair in PEM format using ``openssl``:
 
 .. code-block:: shell
 
     cd /etc/carta
     openssl genrsa -out carta_private.pem 4096
     openssl rsa -in carta_private.pem -outform PEM -pubout -out carta_public.pem
 
+The access tokens use a public/private keypair whether using LDAP, PAM, or OIDC authentication.  However, when using OIDC authentication an additional symmetric encryption key is required for the refresh tokens. (When using LDAP or PAM authentication the same public/private keypair as used with the access tokens is also used for the refresh tokens).  If using the default encryption algorithm, you can again use ``openssl`` to generate the needed key:
```

Comment:
Suggestion: "A public/private keypair is used to authenticate tokens. OIDC authentication requires an additional symmetric encryption key for refresh tokens. LDAP or PAM authentication uses the same public/private keypair both for access tokens and for refresh tokens.  If you use the default encryption algorithm, you can again use ``openssl`` to generate the needed key:"

Reply:
Made a one-word tweak to your suggested revision, but think I can mark this one as resolved.